### PR TITLE
first page creation

### PR DESCRIPTION
**Pull request by dhorelik**
- [x] Close the appearing pop up
- [x] Click on "Add the first page" to open the modal
- [x] Follow the wizard to create the first page
- [x] Add example content during the wizard creation

### CODE REVIEW

File: testserver.py [hunk 1]
```diff
@@ -46,7 +46,10 @@ def noop_gettext(s):
             'public': True,
             'hide_untranslated': False,
         }
-    }
+    },
+    INSTALLED_APPS=[
```
Comment (FinalAngel) [inline testserver.py]: shouldn't this be += ?

Comment (mikek) [inline testserver.py]: @FinalAngel `INSTALLED_APPS` is one of the special settings and it's contents are appended to the defaults: http://djangocms-helper.readthedocs.org/en/develop/settings.html#special-settings.

Comment (FinalAngel) [inline testserver.py]: alright :+1:

**Comment by Brankub** [comment]
:+1:

**Pull request merged**